target/linux/ramips/image/mt7620.mk

```diff
@@ -878,6 +901,28 @@ define Device/tplink_archer-c20i
 endef
 TARGET_DEVICES += tplink_archer-c20i
 
+define Device/tplink_re200-v1
+  SOC := mt7620a
+  DEVICE_VENDOR := TP-Link
+  DEVICE_PACKAGES := kmod-mt76x0e
+  DEVICE_MODEL := RE200
+  DEVICE_VARIANT := v1
+  IMAGES+= factory.bin
+  IMAGE/sysupgrade.bin := mktplinkfw sysupgrade -e -O
+  IMAGE/factory.bin := mktplinkfw factory -e -O
+  IMAGE_SIZE := 7936k
+  KERNEL_ENTRY := 0x8000c310
+  KERNEL_LOADADDR := 0x80000000
+  KERNEL := $(KERNEL_DTB)
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | tplink-v1-header -e -O
+  TPLINK_BOARD_ID := RE200-V1
```

Comment:
this is not needed as the device doesn't use tplink-safeloader

Reply:
There are four lines displayed, what exactly do you mean?

Reply:
the one directly above.

Reply:
OK, done.